tools/IceRpc.Protobuf.Tools/IceRpc.Protobuf.Tools.csproj

```diff
@@ -73,27 +74,60 @@
       </None>
    </ItemGroup>
    <Target Name="CopyGoogleProtobufTools" BeforeTargets="CoreCompile">
+
+      <!-- Download and extract macOS and Linux ARM64 binaries from protobuf release package. The compilers for ARM64
+           are not included with Google.Protobuf NuGet package.
+      -->
+      <DownloadFile
+         SourceUrl="https://github.com/protocolbuffers/protobuf/releases/download/v$(ProtobufVersion)/protoc-$(ProtobufVersion)-osx-aarch_64.zip"
+         DestinationFolder="$(MSBuildThisFileDirectory)obj"
+         Condition="!Exists('$(MSBuildThisFileDirectory)obj/protoc-$(ProtobufVersion)-osx-aarch_64.zip')"/>
+
+      <DownloadFile
+         SourceUrl="https://github.com/protocolbuffers/protobuf/releases/download/v$(ProtobufVersion)/protoc-$(ProtobufVersion)-linux-aarch_64.zip"
+         DestinationFolder="$(MSBuildThisFileDirectory)obj" />
+
+      <!-- We cannot use MSBuild Unzip task because it doesn't preserve the executable permissions -->
+      <MakeDir Directories="$(MSBuildThisFileDirectory)obj/protoc-$(ProtobufVersion)-osx-aarch_64"
+         Condition="!Exists('$(MSBuildThisFileDirectory)obj/protoc-$(ProtobufVersion)-osx-aarch_64')"/>
+      <Exec Command="tar -xf $(MSBuildThisFileDirectory)obj/protoc-$(ProtobufVersion)-osx-aarch_64.zip -C $(MSBuildThisFileDirectory)obj/protoc-$(ProtobufVersion)-osx-aarch_64"
+            Condition="!Exists('$(MSBuildThisFileDirectory)obj/protoc-$(ProtobufVersion)-osx-aarch_64/bin/protoc')"/>
+
+      <MakeDir Directories="$(MSBuildThisFileDirectory)obj/protoc-$(ProtobufVersion)-linux-aarch_64"
+         Condition="!Exists('$(MSBuildThisFileDirectory)obj/protoc-$(ProtobufVersion)-linux-aarch_64')"/>
+      <Exec Command="tar -xf $(MSBuildThisFileDirectory)obj/protoc-$(ProtobufVersion)-linux-aarch_64.zip -C $(MSBuildThisFileDirectory)obj/protoc-$(ProtobufVersion)-linux-aarch_64"
+            Condition="!Exists('$(MSBuildThisFileDirectory)obj/protoc-$(ProtobufVersion)-linux-aarch_64/bin/protoc')"/>
+
       <ItemGroup>
          <GoogleProtobufFiles Include="$(protoc_tools)/google/protobuf/*.*"/>
-         <GoogleLinuxProtoc Include="$(protoc_tools)/linux_x64/*.*" />
-         <GoogleWindowsProtoc Include="$(protoc_tools)/windows_x64/*.*" />
-         <GoogleMacOsxProtoc Include="$(protoc_tools)/macosx_x64/*.*" />
       </ItemGroup>
+
       <Copy
             SourceFiles="@(GoogleProtobufFiles)"
             DestinationFolder="$(MSBuildThisFileDirectory)obj/tools/google/protobuf"
             SkipUnchangedFiles="true" />
       <Copy
-            SourceFiles="@(GoogleLinuxProtoc)"
-            DestinationFolder="$(MSBuildThisFileDirectory)obj/tools/linux_x64"
+            SourceFiles="$(protoc_tools)/linux_x64/protoc"
+            DestinationFolder="$(MSBuildThisFileDirectory)obj/tools/linux-x64"
```

Comment:
I would not change the path name from linux_x64 to linux-x64 in a patch release. Also, why change it at all? linux_x64 sounds fine to me.

Reply:
We use linux-x64 in Slice compilers, linux_x64 in Protoc, just for consistency. But this can wait for non patch release.